test/suites-ostree/failed-upgrade.robot

```diff
@@ -30,7 +30,9 @@ Staged Deployment Consistently Fails To Back Up The Data
     TestAgent.Add Action For Next Deployment    every    prevent_backup
```

Comment:
Please take a look at [current version of this file](https://github.com/openshift/microshift/blob/main/test/suites-ostree/failed-upgrade.robot#L31). I think you need to rebase these changes